docs/var.tfvars-doc.md

```diff
@@ -179,12 +179,38 @@ This variable can be used for trying out custom OpenShift install image for deve
 release_image_override     = ""
 ```
 
-These variables specify the ansible playbooks that are used for OpenShift install and post-install customizations.
+These variables specify the ansible playbooks that are used for OpenShift install and post-install customizations. If the URL ends with a file name extension .zip, then it is assumed that it points to a HTTP/HTTPS server and curl/unzip will be used to extract the package. URLs without ending with .zip are recognized as GitHub repositories and git clone && git checkout are used.
+`Only .zip is supported file format on web servers. The all files must be placed in a subfolder called ocp4-playbooks-master, or ocp4-helpernode-master! This is the default structure, if you download the playbooks from GitHub.com website and uses the "Download as ZIP" function, which is the recommended way. Exception is the help_repo, which is expected to be a tar.gz file format (default when downloading from the original source without manipulation afterwards!`
+Valid options: Requires a URL pointing to the packages/GitHub project.
 ```
+helpernode_repo            = "https://<HTTP SERVER>/ocp4-ansible-modules/ocp4-helpernode-master.zip"
+OR
 helpernode_repo            = "https://github.com/RedHatOfficial/ocp4-helpernode"
 helpernode_tag             = "5eab3db53976bb16be582f2edc2de02f7510050d"
+
+install_playbook_repo      = "https://<HTTP SERVER>/ocp4-ansible-modules/ocp4-playbooks-master.zip"
+OR
 install_playbook_repo      = "https://github.com/ocp-power-automation/ocp4-playbooks"
 install_playbook_tag       = "02a598faa332aa2c3d53e8edd0e840440ff74bd5"
+helm_repo                  = "https://<HTTP SERVER>/python-modules/helm-latest-linux-ppc64le.tar.gz"
+```
+
+If you want to provide the ansible playbooks by your local HTTP server, follow these steps:
+```
+Use your web browser and visit https://github.com/RedHatOfficial/ocp4-helpernode
```

Comment:
We would like to not use mater branch always (used only for preview versions). You can say select the release branch and then click....

Reply:
Updated it like you had mentioned it.